components/Common/Sidebar/SidebarGroup/index.module.css

```
@@ -0,0 +1,24 @@
+.group {
+  @apply flex
+  w-full
+  flex-col
+  gap-2;
+}
+
+.groupName {
+  @apply mb-2
+    px-2
+    text-sm
+    font-semibold
+  text-neutral-800
+  dark:text-neutral-200;
+}
```

```suggestion
.groupName {
  @apply mb-2
    px-2
    text-sm
    font-semibold
    text-neutral-800
    dark:text-neutral-200;
}
```

Missing indentations 👀